From 580ddc15f717b285ca7941268059a21e0e16ad1f Mon Sep 17 00:00:00 2001
From: kboeluek <kboeluek@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 3 Sep 2015 10:05:50 +0000
Subject: [PATCH 190/462] MANTIS 0037084: AEA Receipt Printer Barcode

- removed GPPDevIM689 from build in cpp/makefile.mak
- removed ImageMagick 689 in setenv.bat


git-svn-id: svn://localhost/SelfServiceBuild/trunk@286 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.mak   | 1 -
 SelfServiceBuild/Build/scripts/setenv.bat | 4 ----
 2 files changed, 5 deletions(-)

diff --git a/SelfServiceBuild/Build/cpp/makefile.mak b/SelfServiceBuild/Build/cpp/makefile.mak
index 8273b16..5c934a3 100644
--- a/SelfServiceBuild/Build/cpp/makefile.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.mak
@@ -119,7 +119,6 @@ PLATFORM_DIRECTORIES= \
  Massai\cpp\GPPDevCustomLib \
  Massai\cpp\GPPDev \
  Massai\cpp\GPPDev\windows_IM63 \
- Massai\cpp\GPPDev\windows_IM689 \
  Massai\cpp\Sel \
  Massai\cpp\ATBTest \
  Massai\cpp\crd\projects\HS \
diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index 8b0dbe0..cb25358 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -371,10 +371,6 @@ set DIR_MAGICK633=%DIR_GLOBALSLIB%\imagemagick\6.3.3-4-Q8\win32
 set INC_MAGICK633=-I%DIR_MAGICK633%\inc
 set LIB_MAGICK633=%DIR_MAGICK633%\lib\CORE_RL_magick_.lib %DIR_MAGICK633%\lib\CORE_RL_wand_.lib %DIR_MAGICK633%\lib\X11.lib
 
-set DIR_MAGICK689=%DIR_GLOBALSLIB%\imagemagick\6.8.9-8-Q16\win32
-set INC_MAGICK689=-I%DIR_MAGICK689%\inc
-set LIB_MAGICK689=%DIR_MAGICK689%\lib\CORE_RL_magick_.lib %DIR_MAGICK689%\lib\CORE_RL_wand_.lib %DIR_MAGICK689%\lib\X11.lib
-
 set DIR_QTSDK=%DIR_GLOBALSLIB%\qtsdk\6.0\win32
 set INC_QTSDK=-I%DIR_QTSDK%\cincludes
 set LIB_QTSDK=%DIR_QTSDK%\libraries\qtmlclient.lib
-- 
2.41.0.windows.1

